### ISSUE TITLE
Sonic Boom on Echo Glass freezes world

### ISSUE THREAD

**Issue by BioMasterZap** [FrozenBlock/WilderWild#414]
I was testing v2.4.6 of this mod for Fabric 1.20.4 and when trying out Echo Glass, it seems when a Warden hits it with a sonic boom, the world freezes up before the glass cracks. I have managed to repeat this multiple times, including in an isolated test environment, ad every time it has resulted in me needing to close the game through task manager.

**Steps To Reproduce**
1. Use v2.4.6 for Fabric 1.20.4.
2. Place Echo Glass.
3. Get the Warden to hit it with a Sonic Boom.
4. The game should be non-responsive.

**Expected Behavior**
According to the mod page, it is suppose to block it and crack or something of the sort. Either way, not freeze the world.

**Logs**
https://gist.github.com/BioMasterZap/9e5b45140d12783f2661bb4d27cd05a1

**Additional context**
When checking previous reports, I noticed #404 seemed to experience the same problem.

The glass was in its full state; the Warden wasn't being cooperative enough to test in its other states. My original tests were underground and my following tests were on the surface. Also I did test it first with the Ancient Horn and that just breaks the glass without any freezing.

**Comment by AViewFromTheTop** [comment]
We do not accept issue reports for outdated versions of Wilder Wild, and the logs do not show any crash.

**Comment by BioMasterZap** [comment]
The logs don't show a crash because the server went completely unresponsive, making it unable to further log anything. I had to end task with task manager to get the game to close every single time the bug occurs.

I don't have a profile set up for 1.21.X to test it for you, but it might be worth checking to make sure this issue hasn't persisted to later versions. If it was fixed, none of the changelogs since v2.4.6 mention any fixes relating to Echo Glass. Also, since it is the latest 1.20.4 release, it might be worth noting the bug if it isn't going to be fixed.
